### PR TITLE
Tags blogpost

### PR DESCRIPTION
**Pull request by Lisa-Stubert**

### CODE REVIEW

File: source/projects/tag-analysis/de/index.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+layout: project
+description: Analysis of the most frequently used tags for Berlins Open Data.
+lang: de
```
Comment (ff6347) [inline source/projects/tag-analysis/de/index.md:4]: If this is the english version the set `lang: en`

File: source/projects/tag-analysis/de/index.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+## Visualization 
+
+Using the information from our data analysis, we used the visNetwork function in R to establish a network visualization of our dataset tags. This visualization includes a search option for tags and categories. Nodes, each of which represents a different tag, are weighted to represent its number of total occurrences across all datasets---the bigger the node, the most frequently it appears on the Berlin Open Data Portal. Edges, which are the lines uniting nodes, illustrate how often tags occur in the same dataset using our initial bigram analysis. The graph is also interactive: when clicking on one of the nodes or edges, the associated tag(s) and their category and occurrence appear. 
+
+- **[Try it yourself here!]()**
```
Comment (ff6347) [inline source/projects/tag-analysis/de/index.md:53]: Missing link

File: source/projects/tag-analysis/en/index.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
+---
+layout: project
+description: Analysis of the most frequently used tags for Berlins Open Data.
+lang: en
```
Comment (ff6347) [inline source/projects/tag-analysis/en/index.md:4]: Also `en`?

File: source/projects/tag-analysis/en/index.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+## Visualization 
+
+Using the information from our data analysis, we used the visNetwork function in R to establish a network visualization of our dataset tags. This visualization includes a search option for tags and categories. Nodes, each of which represents a different tag, are weighted to represent its number of total occurrences across all datasets---the bigger the node, the most frequently it appears on the Berlin Open Data Portal. Edges, which are the lines uniting nodes, illustrate how often tags occur in the same dataset using our initial bigram analysis. The graph is also interactive: when clicking on one of the nodes or edges, the associated tag(s) and their category and occurrence appear. 
+
+- **[Try it yourself here!]()**
```
Comment (ff6347) [inline source/projects/tag-analysis/en/index.md:53]: missing link

**Review by ff6347 — CHANGES_REQUESTED** [review]

**Comment by ff6347** [comment]
The diagram looks pretty huge.

**Comment by Lisa-Stubert** [comment]
needs final content review. pls do not merge

**Comment by ff6347** [comment]
> needs final content review. pls do not merge

Maybe we set it to draft state.

File: source/projects/tag-analysis/en/index.md (hunk 3 above)
Comment (Lisa-Stubert) [reply source/projects/tag-analysis/en/index.md:4]: en content was added now

File: source/projects/tag-analysis/en/index.md (hunk 4 above)
Comment (Lisa-Stubert) [reply source/projects/tag-analysis/en/index.md]: link is working now

File: source/projects/tag-analysis/de/index.md (hunk 2 above)
Comment (Lisa-Stubert) [reply source/projects/tag-analysis/de/index.md]: link is working now

**Comment by Lisa-Stubert** [comment]
@ff6347 @dnsos this pull request is ready for merge now

**Review by ff6347 — APPROVED** [review]
LGTM

**Review by dnsos — APPROVED** [review]
Merging this now.

**Pull request merged**